app/views/login-view.js

```diff
@@ -0,0 +1,65 @@
+import { UserModel } from '../models/user-model';
+import Utils from '../utils.js';
+
+const LoginView = Backbone.View.extend({
+  inputvalues: {
+    username: 'default',
+    password: 'default',
+  },
+  template: $('#template-Login').html(),
+  render() {
+    this.$el.html(this.template);
+    return this;
+  },
+  events: {
+    'click #submitSignIn': 'handleLogin',
+  },
+  handleLogin(event) {
+    this.inputvalues.username = document.forms.signInForm.username.value;
+    this.inputvalues.password = document.forms.signInForm.password.value;
+    event.preventDefault();
+    this.sendLoginPost(this.inputvalues);
+  },
+  sendLoginPost(data) {
+    $.ajax({
+      type: 'POST',
+      url: 'http://localhost:3000/auth',
+      data: JSON.stringify(data),
+      dataType: 'json',
+      contentType: 'application/json',
+    }).done((resp) => {
+      window.LoggedInUser.token = resp;
+      this.getPreferences();
+    });
+  },
+
+  getPreferences() {
+    $.ajax({
+      type: 'GET',
+      url: 'http://localhost:3000/preferences',
+      headers: {
+        'x-token': window.LoggedInUser.token,
+      },
+    }).done((resp) => {
+      const jsonObj = JSON.parse(resp);
+      window.LoggedInUser.fullname = jsonObj.gsx$fullname.$t;
+      Utils.setHeaderBackground(jsonObj.gsx$background.$t);
+    });
+  },
+// Logout trebuie mutata !!! No rage :)
+  logout() {
+    $.ajax({
```

Comment:
view-ul acesta are prea multe responsabilităti:

- afişează un formular
- autentifică cu serverul
- aduce preferinţele unui utilizator
- salveaza date pe window

e o soluţie simplă să ţii current user pe window dar trebuie să reorganizezi codul şi logica; poţi crea un model de User pe care îl instanţiezi pre-login, care are ca atribute default username si password goale, care are funcţii de log in, log out şi fetch şi când face login vei face fetch iar apoi îl vei seta ca şi window.currentUser

sunt mai multe abordări la a avea current user disponibil, majoritatea mai bune decât pe window, dar pentru problema ta e OK rezolvarea descrisă sus